pastas/stats/tests.py

```diff
@@ -7,17 +7,21 @@
 from logging import getLogger
 
 from numpy import arange, cumsum, finfo, median, nan, sqrt, zeros
-from pandas import DataFrame, date_range, infer_freq
+from pandas import Series, DataFrame, date_range, infer_freq
 from pastas.stats.core import acf as get_acf
 from pastas.utils import _get_time_offset, get_equidistant_series
 from scipy.stats import chi2, norm, normaltest, shapiro
 
+# Type Hinting
+from typing import Tuple
+
+
 logger = getLogger(__name__)
 __all__ = ["durbin_watson", "ljung_box", "runs_test", "stoffer_toloi",
            "diagnostics", "plot_acf", "plot_diagnostics"]
 
 
-def durbin_watson(series=None):
+def durbin_watson(series: Series = None) -> float:
```

Comment:
Optional[Series]?

Reply:
Wait, why is series optional to begin with?

Reply:
Agreed that None should not be optional to begin with.